pixel.js

```diff
@@ -132,19 +136,32 @@ export default class PixelPlugin
                 if (lastLayer !== this.selectedLayer && this.mousePressed)
                     this.keyboardChangingLayers = true;
             }
-
             if (key === shiftKey)
             {
-                this.shiftDown = true;
-            }
-            else
-            {
                 this.shiftDown = false;
             }
         };
         document.addEventListener("keyup", handle);
+        return handle;
 
+    }
+
+    //Deals with keyboard button down press
+    subscribeToKeyboardPress()
+    {
+        let handle = (e) =>
+        {
+            const shiftKey = 16;
```

Comment:
Constants in Diva are all non-uppercased. Where is the line drawn for adopting the prevailing style? 🚒